src/components/ScheduleFormModal.vue

```diff
@@ -23,6 +23,26 @@
 
     <FlowRunJobVariableOverridesLabeledInput v-if="can.access.deploymentScheduleFlowRunInfraOverrides" v-model="internalJobVariables" />
 
+    <p-label
+      label="Max active runs"
+      :message="maxActiveRunsErrorMessage"
+      :state="maxActiveRunsState"
+      class="interval-schedule-form__column--span-3"
+    >
+      <p-number-input v-model="internalMaxActive" min="0" :state="maxActiveRunsState" />
+    </p-label>
```

Comment:
I don't think there's a way to nullify this value rn - any schedule created will have a value for this, defaulting to 0. I'd recommend we take page from the schemas work and provide a way to omit this value.